tequilapi/endpoints/service.go

```diff
@@ -70,7 +72,27 @@ func NewServiceEndpoint(serviceManager ServiceManager, optionsParser map[string]
 //     schema:
 //       "$ref": "#/definitions/APIError"
 func (se *ServiceEndpoint) ServiceList(c *gin.Context) {
-	instances := se.serviceManager.List()
+	includeAll := false
+	includeAllStr := c.Request.URL.Query().Get("include_all")
```

Comment:
There is `c.Get` or `c.GetBool` wouldn't those do the same but better?

Reply:
as I understand those methods doesn't fetch data from query params. it looks inside `c.Keys` map[string]interface{}

Reply:
`c.Query("include_all")` ?